site/docs/how-tos/operator.md

```diff
@@ -0,0 +1,309 @@
+...
+```
+
+Download and unpack Hazelcast Jet distribution from [Jet
+Website](https://jet-start.sh/download) to use sample job packaged with it.
+
+In a different terminal window, unpack and navigate to the distribution
+folder with the commands below:
+
+```bash
+tar xf hazelcast-jet-4.1.tar.gz
+cd hazelcast-jet-4.1/
+```
+
+Verify that CLI can connect to the cluster with the command below:
+
+```bash
+$ bin/jet cluster
+State: ACTIVE
+Version: 4.1
+Size: 2
+
+ADDRESS                  UUID
+[10.44.1.11]:5701        a720a93f-418a-4656-82a9-ae69eaeba99d
+[10.44.2.9]:5701         5ac07dd4-e631-444e-a9ab-5e5faf1cd617
+```
+
+Then you can submit the example job from the distribution package with
+the command below:
+
+```bash
+$ bin/jet submit examples/hello-world.jar
+Submitting JAR 'examples/hello-world.jar' with arguments []
+Top 10 random numbers in the latest window:
+ 1. 9,192,248,499,877,110,770
+ 2. 9,137,762,581,670,149,663
+ 3. 8,919,106,540,516,409,274
+ 4. 8,802,486,215,849,765,726
+ 5. 7,618,430,691,765,141,863
+ 6. 7,346,532,478,235,724,392
+ 7. 7,097,603,655,485,784,276
+ 8. 7,077,059,954,496,239,921
+ 9. 6,489,569,388,718,300,773
+ 10. 6,477,025,315,319,514,141
+...
+```
+
+That's it, you've successfully deployed a job from your local machine to
+the Hazelcast Jet cluster running inside Kubernetes
+environment.
+
+## Connect to Hazelcast Jet Management Center
+
+Run the following command to get Hazelcast Jet Management Center service
+details:
+
+```bash
+$ kubectl get svc
+NAME                                  TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)                        AGE
+jet-hazelcast-jet                     ClusterIP      None           <none>          5701/TCP                       54m
+jet-hazelcast-jet-management-center   LoadBalancer   10.47.244.94   34.72.142.219   8081:32476/TCP,443:32535/TCP   54m
+kubernetes                            ClusterIP      10.47.240.1    <none>          443/TCP                        2d23h
+```
+
+To connect to Hazelcast Jet Management Center, you can use `EXTERNAL-IP`
+and open your browser at: `http://<EXTERNAL-IP>:8081`.
+
+> If you are running on a Minikube environment which does not have
+> Load Balancer support out of the box and it will show the `EXTERNAL-IP`
+>as `pending`.
+>
+>To solve this problem, on a separate terminal window, run the following
+>command:
+>
+>```bash
+>minikube tunnel
+>```
+>
+>It will ask for a password and expose service to the host operating
+>system.
+>
+>When asked for service details with the following:
+>
+>```bash
+>kubectl get svc
+>```
+
+It should show the `EXTERNAL-IP` and Hazelcast Jet Management Center
+should be accessible at  `http://<EXTERNAL-IP>:8081`.
+
+You should see the login page at `http://<EXTERNAL-IP>:8081` like below and
+use `admin` as both username and password to login.
+
+![Hazelcast Jet Management Center Login](assets/mc-login.png)
+
+You can see various details regarding cluster and the job in the
```

Comment:
I can't, just get the (expected) message `Free trial is limited to a single node. Have you entered your license key ?  Learn how to fix it.`

we should include a link to get a trial and add a step to setup the key.

Reply:
Sorry, it's actually in the `Creating a Hazelcast Jet Cluster` section, I just filter out the yellow stuff :-)